specifyweb/frontend/js_src/lib/components/WebLinks/index.tsx

```diff
@@ -154,6 +155,7 @@ export function WebLinkField({
             }}
           >
             {image}
+            <div className="text-transform-none">{localized(url ?? '')}</div>
           </Component>
```

Comment:
I was [suggesting](https://github.com/specify/specify7/issues/4335#issuecomment-1871684164) we put the URL outside the button, not inside:

![Screenshot 2024-01-02 at 18 36 35](https://github.com/specify/specify7/assets/40512816/b9eea057-7cd9-413b-ac5f-f8fe7594ffa8)

Reasons:
- URL can be selected without triggering the button
- The button looks the way it would look like on the form
- Better for accessibility (having a URL as button label is going to sound bad in the screen reader)

Reply:
Also, I just realized that this component is the same on that is used on the form - we don't want the weblink on the form to print the URL

So, you should add something like this in the component to check if we are in the web link editor:

```ts
  const isInEditor = React.useContext(WebLinksContext) !== undefined;
```

and then only print the URL preview if we are in the weblink editor